google/cloud/forseti/services/dao.py

```diff
@@ -86,10 +87,16 @@ class Model(MODEL_BASE):
     created_at = Column(DateTime)
     etag_seed = Column(String(32), nullable=False)
     message = Column(Text())
-    warnings = Column(Text())
+    warnings = Column(MEDIUMTEXT())
```

Comment:
Can the warnings column in the inventory index schema also be updated to MEDIUMTEXT?

Reply:
Yup, let me do that. It shouldn't cause any problem on compatibility. But from "sqlalchemy.dialects.mysql import MEDIUMTEXT" is not compatible with test. That is what bothers me now.

Reply:
Yeah, that is a problem. We the code to transparently support sqlite or mysql or DBs.